system/handlers/admin/DataManager.cfc

```diff
@@ -200,11 +200,19 @@
 			_checkPermission( argumentCollection=arguments, key="read", object=objectName );
 
 			for( var filterByField in filterByFields ) {
-				if( !isEmpty( rc[filterByField] ?: "" ) ){
-					extraFilters.append({
-						  filter       = "#filterByField# = :#filterByField#"
-						, filterParams = { "#filterByField#" = rc[filterByField] }
-					})
+				filterValue = rc[filterByField] ?: "";
+				if( !isEmpty( filterValue ) ){
+					if ( isValid( "UUID", listFirst( filterValue ) ) ) {
```

Comment:
Can we be sure that we only want to behave as an array list when first item is a UUID? Probably fine most of the time, but at some point we may well have natural keys for PK (e.g. system generated data like system emails in email centre have PKs that are human readable from config).

What's the reasoning behind this check?